appveyor.yml

```diff
@@ -7,6 +7,12 @@
   build_script:
     - nuget restore ./ECMA2Yaml/ECMA2Yaml.sln
     - msbuild ./ECMA2Yaml/ECMA2Yaml.sln /p:Configuration=Release
+    - %~dp0ECMA2Yaml/ECMA2Yaml/bin/Release/ECMA2Yaml.exe -s %~dp0test/xml -o %~dp0test/xml/yml_MREF_After -f
```

Comment:
I'm not sure if `%~dp0` can work here because appveyor could be just running these commands line by line instead of putting them in a .bat file. You can just change to something like:

`./ECMA2Yaml/ECMA2Yaml/bin/Release/ECMA2Yaml.exe -s ./test/xml -o ./test/xml/yml_MREF_After -f`

Reply:
`%` is also breaking the yml file, you can use http://www.yamllint.com/ to check if your yaml is in valid format.